pkg/pgmodel/querier/query_builder_samples.go

```diff
@@ -81,9 +81,9 @@ const (
 	     %[3]s`
 
 	/* optimized for no clauses besides __name__
-	   uses a inner join without a lateral to allow for better parallel execution
+	   uses an inner join without a lateral to allow for better parallel execution
```

Comment:
Can we update this one to proper Go style too? Full sentences and not multi-line block.

Reply:
👍